roles/keystone/tasks/keystone-idp.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: setup keystone idp metadata
+  shell: keystone-manage saml_idp_metadata > /etc/keystone/keystone_idp_metadata.xml
+  when: keystone.keystone_identity_provider.enabled|bool
+  tags: k2k-federation
```

Comment:
I'm going to leave it in there. I'm not sure how to check if the saml signing cert/key changed or if the keystone conf has changed or if the file does not exist, then run the command.